pkg/modulewriter/tfwriter.go

```diff
@@ -103,8 +103,8 @@ func writeOutputs(
 				desc = fmt.Sprintf("Generated output from module '%s'", mod.ID)
 			}
 			blockBody.SetAttributeValue("description", cty.StringVal(desc))
-			value := fmt.Sprintf("((module.%s.%s))", mod.ID, output.Name)
-			blockBody.SetAttributeValue("value", cty.StringVal(value))
```

Comment:
To be aligned with rest of the code we could write:
```
value := cty.StringVal(fmt.Sprintf("((module.%s.%s))", mod.ID, output.Name))
blockBody.SetAttributeRaw("value", TokensForValue(value))
```
But this code will be likely changed soon, I will add tests when I touch it next time.

Reply:
Offline discussion: this works and may be modified when unit test added to test for this regression